functions.php

```diff
@@ -68,6 +68,47 @@ function course_theme_init() {
 			'label' => __( 'Button', 'course' ),
 		)
 	);
+	register_block_style(
```

Comment:
Why do we need block styles for this?

Reply:
Thanks for taking a look @scruffian, I've removed the block style [here](https://github.com/Automattic/sensei-theme/pull/44/commits/a3cde4a814d1a42e2c865715a54c7a9d9a3832f5)